README.md

```diff
@@ -132,12 +132,12 @@ canned_responses: true
 
 ### Adult age
 
-Visit requests are limited to a maximum of 3 adults. Visiting areas have 3 seats for visitors and one for the prisoner. Children are expect to sit on the laps of adults.
+Visit requests are limited to a maximum of 3 "adults" (18 years old and over). The adult age can lessened to restrict the amount of visitors over that age.
```

Comment:
`The adult age can lessened` :arrow_right: `The adult age can be reduced`, maybe?